pages/workspaceLogs/workspaceLogs.html.js

```diff
@@ -0,0 +1,239 @@
+            <table class="table table-sm">
+              <thead>
+                <th>Version</th>
+                <th>Created</th>
+                <th>Actions</th>
+              </thead>
+              <tbody>
+                ${uniqueVersions.map((version) => {
+                  const logsUrl = `${resLocals.urlPrefix}/workspace/${resLocals.workspace_id}/logs/version/${version.version}`;
+                  return html`
+                    <tr>
+                      <td>${version.version}</td>
+                      <td>${version.date_formatted}</td>
+                      <td>
+                        <a href="${logsUrl}"> View detailed logs </a>
+                      </td>
+                    </tr>
+                  `;
+                })}
+              </tbody>
+            </table>
+          </div>
+
+          <h2>History</h2>
+          ${WorkspaceLogsTable({ workspaceLogs })}
+        </div>
+      </body>
+    </html>
+  `.toString();
+};
+
+const WorkspaceVersionLogs = ({
+  version,
+  workspaceLogs,
+  containerLogsEnabled,
+  containerLogsExpired,
+  resLocals,
+}) => {
+  return html`
+    <!DOCTYPE html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../partials/head'); %>", {
+          ...resLocals,
+          pageTitle: 'Workspace logs',
+        })}
+      </head>
+      <body>
+        ${renderEjs(__filename, "<%- include('../partials/navbar'); %>", {
+          ...resLocals,
+          navPage: 'plain',
+        })}
+
+        <div id="content" class="container mb-4">
+          <h1 class="mb-4">Workspace version logs</h1>
+
+          <h2>Container logs</h2>
+          ${containerLogsEnabled && !containerLogsExpired
+            ? html`
+                <div
+                  id="js-container-logs"
+                  class="mb-3"
+                  data-logs-endpoint="${resLocals.urlPrefix}/workspace/${resLocals.workspace_id}/logs/version/${version}/container_logs"
+                >
+                  <pre
+                    class="bg-dark rounded-top text-white p-3 mb-0"
+                  ><code class="js-output"></code></pre>
+                  <div class="bg-secondary rounded-bottom">
+                    <button type="button" class="btn btn-light m-3 js-load-more-logs">
+                      <span
+                        class="spinner-border spinner-border-sm js-loading-spinner"
+                        role="status"
+                        aria-hidden="true"
+                        hidden
+                      ></span>
+                      <span class="js-button-message">Load more logs</span>
+                    </button>
+                  </div>
+                </div>
+              `
+            : html`
+                <div class="bg-dark py-5 px-2 mb-3 rounded text-white text-center text-monospace">
+                  <div class="mb-2">
+                    <i class="fa fa-calendar fa-2xl" aria-hidden="true"></i>
```

Comment:
TODO: pick a better icon for this when container logs are disabled - maybe [`fa-ban`](https://fontawesome.com/icons/ban?s=solid&f=classic)?

Reply:
Ideas:
https://icons.getbootstrap.com/icons/body-text/
https://icons.getbootstrap.com/icons/card-text/
https://icons.getbootstrap.com/icons/file-text/
https://icons.getbootstrap.com/icons/list-columns-reverse/
https://icons.getbootstrap.com/icons/receipt/

Reply:
I like `fa-ban` for this.

Reply:
Done in 8c73f53.